chargeOclock/Source/Other/NetworkAPI.swift

```diff
@@ -21,11 +21,11 @@ extension NetworkAPI {
 	}
 
 	func create(client: Client, completion: @escaping NetworkCompletion) {
-		performPostRequest(to: "clients", json: client.toJSON(), completion: completion)
+		performPostRequest(to: "clients", data: client.toData(), completion: completion)
```

Comment:
the serialization of the JSON dictionary, representing the Client object, should be done in this method, and passed on as an argument